stage0/interface.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2016 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//+build linux
```

Comment:
Actually, I think the idea was being able to build rkt on non-Linux systems to fetch images and so (not really sure that it actually works).

Hence, we only build the contents of `stage0/` on Linux because that's execution code, while the fetch code lives in `rkt/`, so we should keep `+build linux`